addons/mail/static/src/models/time_user/time_user.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { insertAndReplace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'TimeUser',
```

Comment:
name to brainstorm

Reply:
Name's strange at first, but when I see what it models I think it's fine.
I think we should take the habit of adding small docstring above all our models.
Here the name feels off because we don't know how to read it.
But it would be totally fine if it says in plain English _"This models records that rely on Time intervals"_: we know it focuses on `Record` <=> `Time` relationship, and it's about `interval` in `Time`, so we get that we should read it as `User of Time`.